### PR TITLE
COMMERCE-9405 Add catalog view permission to Discount Manager

### PR DESCRIPTION
**Pull request by smottal**
https://issues.liferay.com/browse/COMMERCE-9405

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Review by luca-pellizzon — APPROVED** [review]
Not sure about the Company Scope, but I see that all of the permissions are imported like so...

**Comment by luca-pellizzon** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-account-service/bnd.bnd [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Commerce Account Service
 Bundle-SymbolicName: com.liferay.commerce.account.service
 Bundle-Version: 4.0.58
-Liferay-Require-SchemaVersion: 10.1.0
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-account-service/bnd.bnd:4]: This one should be inside a dedicated commit (baseline)

**Review by luca-pellizzon — CHANGES_REQUESTED** [review]
Need to run baseline

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by victoroscuro** [comment]
ci:test:commerce

**Comment by lorenzolapescara** [comment]
@smottal this PR has conflicts, can you please check them? Thanks!

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:stop

**Comment by smottal** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
ci:test:commerce

**Comment by lorenzolapescara** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
@smottal I re-sent this at #3224, can you close this PR? Thanks!

**Pull request closed without merging**